frontend/src/views/admin/DashboardPage/DashboardPage.tsx

```diff
@@ -56,7 +68,15 @@ export const AdminDashboardPage = () => {
       signUpMode: config.allowSignUp ? SignUpModes.Anyone : SignUpModes.Disabled,
       allowedSignUpDomain: config.allowedSignUpDomain,
       trustSamlEmails: config.trustSamlEmails,
-      trustLdapEmails: config.trustLdapEmails
+      trustLdapEmails: config.trustLdapEmails,
```

Comment:
As its a tab and two independent form i would recommend to split this into two components.

Then you can put the form validation required for each tab in there component space independently. Issue i see here is what like one tab has an error the other tab form won't work.